pyproject.toml

```diff
@@ -125,7 +125,6 @@ requires = ["setuptools",
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-include-package-data = true
```

Comment:
This is being discussed separately at https://github.com/astropy/astropy/pull/15422

Reply:
Because of the other pull request it is best not to edit the setting here.